OutOfSchool/OutOfSchool.WebApi.Tests/Controllers/ApplicationControllerTests.cs

```diff
@@ -316,12 +317,78 @@ public async Task GetByWorkshopId_WhenProviderHasNoApplications_ShouldReturnNoCo
         result.StatusCode.Should().Be(StatusCodes.Status204NoContent);
     }
 
+    [Test]
+    public async Task GetPendingApplicationsByProviderId_WhenProviderDoesNotHaveAnyApplications_ShouldReturnNoContent()
+    {
+        // Arrange
+        var newProviderStandard = new ProviderDto { Id = providerId, UserId = userId };
+
+        httpContext.Setup(c => c.User.IsInRole("provider")).Returns(true);
+
+        providerService.Setup(s => s.GetById(providerId)).ReturnsAsync(newProviderStandard);
+
+        var app1 = applications.Where(a => a.Workshop.ProviderId == providerId).ToList();
```

Comment:
(Code style) Name `app1`..., oh well. Please avoid usage of number suffix in variables name, especially when domain doesn't require it (and there is no any "app2", "app3" ...)

Reply:
Fixed. Now it is providerApplications.